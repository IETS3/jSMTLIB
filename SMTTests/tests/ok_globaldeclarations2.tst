; checks global declarations
(set-logic QF_UF)
(declare-fun a () Bool)
(reset-assertions)
(declare-fun a () Bool)
(check-sat)  ; sat
