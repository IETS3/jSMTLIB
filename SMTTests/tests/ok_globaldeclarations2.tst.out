success
success
success
success
sat
